src/routes/api/v1/priceChange/index.ts

```diff
@@ -35,7 +64,33 @@ const router = Router()
  *
  *
  * */
-router.get(
+/**
+ * @api {post} /pricechange Get price change for token over time duration
+ * @apiName GetTokenPriceChangeOverDuration
+ * @apiGroup PriceChange
+ *
+ * @apiParam {String} tokenAddress The currency address
+ * @apiParam {Object} duration The duration object to calculate the price change over the timeframe
+ * duration should be passed as an object according to https://day.js.org/docs/en/durations/creating
+ * for example duration of {days: 1} means a duration of one day
+ *
+ * @apiSuccess {String} priceChange The price change ratio of the token
+ * @apiSuccess {String} currentPrice The current price of the token
+ * @apiSuccess {Object} previousPrice The previous price of the token
+ *
+ * @apiSuccessExample {json} Success-Response:
+ *
+ *  {
+ *      "data": {
+ *          "priceChange": "4.761727644165598",
+ *          "currentPrice": "3760.8426158182515",
+ *          "previousPrice": "3589.901293526158"
+ *      }
+ *  }
+ *
+ *
+ * */
+router.post(
```

Comment:
Hmmmmm, maybe we append a path to the route, wdyt?

Reply:
And i also see we are using the same handler twice

Reply:
And above this we have two docs for the route, maybe the get doc shouldn't be there

Reply:
deleted the double docs. I think now it's cool cause:
- two controllers can handle same route if it's the same logic (core reuse)
- there's two ways to do the priceChange query, via GET (the simple way), via POST with optional arguments. It's pretty common to do a query with POST if it's not a CRUD API endpoint